src/components/NoConnectionIndicator/NoConnectionIndicator.tsx

```diff
@@ -0,0 +1,35 @@
+import { ConnectionStatus } from '@/hooks'
+import { Spinner } from '@/shared/components'
+import { transitions } from '@/shared/theme'
+import React from 'react'
+import { CSSTransition } from 'react-transition-group'
+import { IndicatorWrapper, StyledSubTitle, StyledTitle, TextWrapper } from './NoConnectionIndicator.style'
+
+export type NoConnectionIndicatorProps = {
+  connectionStatus: ConnectionStatus
+}
+
+const NoConnectionIndicator: React.FC<NoConnectionIndicatorProps> = ({ connectionStatus }) => {
+  return (
+    <CSSTransition
+      in={connectionStatus === 'no-internet' || connectionStatus === 'disconnected'}
+      timeout={200}
+      classNames={transitions.names.fade}
+      mountOnEnter
+      unmountOnExit
+    >
+      <IndicatorWrapper>
+        <Spinner size="small" />
```

Comment:
I believe Spinner will need some refactor to cover different color variations. The grey part in designs is lighter